scrape.py

```diff
@@ -124,7 +124,7 @@ def __repr__(self):
 
 
 def sanitize_instance_type(instance_type):
-    """Typos and other bad data are common in the instance type colums for some reason"""
+    """Typos and other bad data are common in the instance type columns for some reason"""
```

Comment:
Having a typo in a comment about typos might have been intentional to prove a point 😁